packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -228,16 +228,16 @@ export class ZoweVsCodeExtension {
         const baseProfile = await cache.fetchBaseProfile(serviceProfile.name);
         if (baseProfile) {
             const tokenType =
-                serviceProfile.profile.tokenType ??
                 baseProfile.profile.tokenType ??
+                serviceProfile.profile.tokenType ??
```

Comment:
Does this mean that extenders will have to check to see if there is a base profile with a tokenValue, temporarily remove it from memory, call this logout method, and then add the baseprofile back?

If that's true... would it make sense to change this with a boolean (optional) parameter passed in to this function. that way they can easily toggle this behavior.